defaults/config.js

```diff
@@ -194,7 +194,15 @@ module.exports = {
 		// @default  6697
 		//
 		port: 6697,
-
+		
+		//
+		// Encoding
+		//
+		// @type     string
```

Comment:
Could you list the examples of possible encodings here? And the proper formatting for them?